R/cookies.R

```diff
@@ -1,10 +1,55 @@
 #' dfe_cookie_script
 #'
+#' Calls in JavaScript dependencies to the shiny app. Function should be placed
```

Comment:
Can we add a sentence about what the JavaScript dependencies do?

Reply:
Have attempted one!